doc/topics/releases/nitrogen.rst

```diff
@@ -220,6 +223,37 @@ release cycle (two major releases after this one), those who are using the
 :ref:`netapi module <all-netapi-modules>`) are encouraged to update their code
 to use ``tgt_type``.
 
+``full_return`` Argument in ``LocalClient`` and ``RunnerClient``
+----------------------------------------------------------------
+
+An ``full_return`` argument has been added to the ``cmd`` and ``cmd_sync``
+methods in ``LocalClient`` and ``RunnerClient`` which causes the return data
+structure to include job meta data such as ``ret_code`` and ``success``.
```

Comment:
Yikes, thank you for the careful eye!

Fixed to `retcode`. And you're right there is no `success` -- I'm surprised there isn't a 1-to-1 between job returns on the CLI and on the event bus. 😕